redwood-treehouse-lazylayout-paging/src/commonMain/kotlin/app/cash/redwood/treehouse/lazylayout/paging/LazyPagingItems.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.treehouse.lazylayout.paging
+
+import androidx.compose.runtime.Composable
+import app.cash.paging.compose.LazyPagingItems
+import app.cash.redwood.treehouse.lazylayout.compose.LazyListScope
+
+public fun <T : Any> LazyListScope.items(
+  items: LazyPagingItems<T>,
+  itemContent: @Composable (value: T?) -> Unit,
+) {
+  items(
+    count = items.itemCount,
+  ) { index ->
+    itemContent(items[index])
+  }
+}
+
+public fun <T : Any> LazyListScope.itemsIndexed(
+  items: LazyPagingItems<T>,
+  itemContent: @Composable (index: Int, value: T?) -> Unit,
+) {
+  items(
+    count = items.itemCount,
+  ) { index ->
+    itemContent(index, items[index])
+  }
+}
```

Comment:
This is essentially a Redwood-ified version of the Compose **_UI_** parts of [`paging-compose` in AndroidX Paging](https://github.com/androidx/androidx/blob/335934d20592a016e7e4306d66c48723f38c03b3/paging/paging-compose/src/main/java/androidx/paging/compose/LazyPagingItems.kt#L285-L360).

The rest of `paging-compose` (which happens to be the rest of the linked file as it's a single file module) exists within Multiplatform Paging as part of `paging-compose-common`.